package.json

```diff
@@ -19,7 +19,7 @@
     "bootstrap-sass": "^3.3.7",
     "eonasdan-bootstrap-datetimepicker": "~4.17.0",
     "jquery": "~3.3.0",
-    "json-editor": "git+https://github.com/azavea/json-editor.git#v10.7.25",
+    "json-editor": "git+https://github.com/jeancochrane/json-editor-community.git#azavea-changes",
```

Comment:
We usually try to avoid having dependencies on personal repositories; would it make sense to fork this over to Azavea?

Reply:
Yeah! I chatted about this briefly with @flibbertigibbet but we never formalized a plan. We thought it would make sense to transfer ownership of `json-editor-community` from me to Azavea. `azavea/json-editor` should probably stay put so that DRIVER can stay pinned to it, but if DRIVER ever gets updated to the new schema editor then we could always remove the old `azavea/json-editor` and make `azavea/json-editor-community` the canonical fork.

Reply:
Okay, that sounds good to me!